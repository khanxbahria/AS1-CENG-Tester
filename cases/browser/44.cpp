#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("vvv", "ii","jj");
Tab B = Tab("hh", "yyy","ff");
Tab C = Tab("lll", "w","cc");
Tab D = Tab("i", "yy","ddd");
Tab E = Tab("v", "r","rrr");
Tab F = Tab("pp", "d","b");
Tab G = Tab("g", "w","fff");
Tab H = Tab("xxx", "hh","u");
Tab I = Tab("ccc", "cc","mmm");
Tab J = Tab("rrr", "kk","h");
Tab K = Tab("hh", "o","aa");
Tab L = Tab("u", "bbb","ttt");
Tab M = Tab("qq", "sss","jjj");
Tab N = Tab("kkk", "ii","z");
Tab O = Tab("kkk", "r","ii");
Tab P = Tab("a", "g","jj");
Tab Q = Tab("g", "ooo","qqq");
Tab R = Tab("aa", "p","o");
Tab S = Tab("yy", "bbb","u");
Tab T = Tab("ooo", "kkk","t");
Tab U = Tab("rr", "f","zz");
Tab V = Tab("qq", "yyy","uuu");
Tab W = Tab("dd", "zzz","uu");
Tab X = Tab("uuu", "hhh","uu");
Tab Y = Tab("o", "ppp","fff");
Tab Z = Tab("bbb", "sss","aaa");
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
c.newTab(A);
d.newTab(B);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.newTab(C);
a.changeActiveTabTo(0);
b.newTab(D);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.newTab(E);
b.moveActiveTabTo(2);
c.newTab(F);
d.newTab(G);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.newTab(H);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.newTab(I);
b.newTab(J);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.newTab(K);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.newTab(L);
d.moveActiveTabTo(1);
a.newTab(M);
b.changeActiveTabTo(1);
c.newTab(N);
d.newTab(O);
a.moveActiveTabTo(4);
b.newTab(P);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.newTab(Q);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.newTab(R);
d.changeActiveTabTo(1);
a.newTab(S);
b.newTab(T);
c.newTab(U);
d.changeActiveTabTo(3);
a.changeActiveTabTo(1);
b.newTab(V);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.newTab(W);
b.newTab(X);
c.changeActiveTabTo(5);
d.newTab(Y);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(4);
b.newTab(Z);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(4);
c.moveActiveTabTo(4);
b.changeActiveTabTo(4);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(4);
d.moveActiveTabTo(1);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
d.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->getWindow(8);
brows->getWindow(8);
brows->closeWindow();
brows->getWindow(5);
brows->switchToWindow(0);
brows->closeEmptyWindows();
brows->switchToWindow(0);
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->getWindow(3);
brows->getWindow(6);
brows->switchToWindow(4);
brows->closeEmptyWindows();
brows->getWindow(8);
brows->closeWindow();
brows->print();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->switchToWindow(4);
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->getWindow(4);
brows->closeEmptyWindows();
brows->switchToWindow(9);
brows->closeEmptyWindows();
brows->closeWindow();
brows->print();
brows->switchToWindow(5);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(8);
brows->closeWindow();
brows->switchToWindow(5);
brows->closeWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->closeEmptyWindows();
brows->switchToWindow(1);
brows->print();
brows->newWindow();
brows->print();
brows->getWindow(0);
brows->closeEmptyWindows();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(4);
brows->newWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(9);
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(2);
brows->switchToWindow(5);
brows->newWindow();
brows->newWindow();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->switchToWindow(5);
brows->switchToWindow(4);
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->switchToWindow(5);
brows->print();
brows->switchToWindow(2);
brows->closeWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->switchToWindow(3);
brows->switchToWindow(3);
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->print();
brows->closeEmptyWindows();
brows->print();
delete brows;
}
	return 0;
}