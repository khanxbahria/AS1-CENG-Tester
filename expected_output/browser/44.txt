The active tab is tab 4
p
aa
o

d
pp
b

bbb
u
ttt

f
rr
zz

ii
vvv
jj

ii
kkk
z





The active tab is tab 4
w
g
fff

o
hh
aa

r
kkk
ii

ppp
o
fff

w
lll
cc





yyy
hh
ff

hh
xxx
u

The active tab is tab 4
w
g
fff

o
hh
aa

r
kkk
ii

ppp
o
fff

w
lll
cc





yyy
hh
ff

hh
xxx
u

The active tab is tab 0




The browser is empty
The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




